### DISCUSSION TITLE
How to properly use `--platform` argument?

### DISCUSSION THREAD

**Discussion by shivangrathore** [python-poetry/poetry#6954] [Q&A] [answered]
What I have to specify? Does it support multiple platforms?

**Comment by neersighted** [comment, marked as answer]
Presumably you are referring to the argument to `add` (your post does not make this clear). `platform` is shorthand for the `sys_platform==` maker and supports the same values. Multiple values or inequality are not supported and require the full marker syntax.

**Reply by shivangrathore** [reply]
Thanks
